### PR TITLE
Privacypolicymodal

### PR DESCRIPTION
**Pull request by johanehr**

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -46,13 +46,13 @@ export default function Home() {
           className="group rounded-lg border border-transparent px-5 py-2 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
         >
           <h2 className={`mb-2 text-xl font-semibold`}>
-            Om projektet{' '}
+            Om projektet och hur påminnelser fungerar{' '}
```
Comment (johanehr) [inline src/app/page.tsx:49]: I think this might be a bit wordy to fit nicely, but if it looks fine, why not! (Haven't checked out the code)

**Review by johanehr — COMMENTED** [review]
Seems like linting is required for the Vercel pre-build to work, BTW

File: src/app/page.tsx (hunk 1 above)
Comment (frankosgit) [reply src/app/page.tsx:49]: Agree!

File: src/app/about/page.tsx [hunk 2]
```diff
@@ -18,28 +20,43 @@ export default function About() {
...
+          Som många andra har vi glömt att flytta bilen inför en städdag, vilket har resulterat i saftiga böter.
           Vi ville snabbt kunna ta reda på vart vi kan låta bilen stå så länge som möjligt, men även ett smidigt sätt att få relevanta påminnelser när det är dags att flytta bilen igen.
         </p>
 
+        {/* notifikationer kommer bli klara när vi lanserad detta   <p className="mt-2">
+            Att bygga en sådan applikation tar en del tid, som ni säkert förstår - men vi tänkte börja
+            med att helt enkelt bygga en enkel karta som dynamiskt visar vart det är lämpligast att parkera just nu baserad på kommande städdagar och tider
+          </p> */}
+
+        <h1 className="text-2xl mt-6">Om Påminnelser och Sekretess</h1>
```
[the remark this answers is not in the document]
Comment (frankosgit) [reply src/app/about/page.tsx:47]: Great! I couldn't remember what we decided but I think that would be optimal, I will update the text to English now.

File: src/app/about/page.tsx [hunk 3]
```diff
@@ -18,28 +20,43 @@ export default function About() {
...
+        <PrivacyPolicyModal />
+        <h1 className="text-2xl mt-6">
+          Tekniken bakom
+        </h1>
+
         <p className="mt-2">
-          Att bygga en sådan applikation tar en del tid, som ni säkert förstår - men vi tänkte börja
-          med att helt enkelt bygga en enkel karta som dynamiskt visar vart det är lämpligast att parkera.
+          Appen är byggd i React, Next.js, TS och Tailwind CSS. Vi använder Google-cloud tasks och SendGrid för notifikationssystemet.
         </p>
```
Comment (frankosgit) [inline src/app/about/page.tsx:60]: or the?

File: src/app/about/page.tsx [hunk 4]
```diff
@@ -18,28 +20,43 @@ export default function About() {
...
 
+        {/* notifikationer kommer bli klara när vi lanserad detta   <p className="mt-2">
+            Att bygga en sådan applikation tar en del tid, som ni säkert förstår - men vi tänkte börja
+            med att helt enkelt bygga en enkel karta som dynamiskt visar vart det är lämpligast att parkera just nu baserad på kommande städdagar och tider
+          </p> */}
+
+        <h1 className="text-2xl mt-6">Om Påminnelser och Sekretess</h1>
+          <p className="mt-4">Så fungerar det:</p>
+          <p className="mt-2">Anpassade Påminnelser:</p> Vårt system skickar ut påminnelser baserat på de tider och platser du anger. Detta gör att du alltid får relevanta påminnelser och aldrig missar en städdag. Du kan själv välja hur långt i förväg du vill bli påmind, och om du vill bli påmind via SMS eller e-post.
```
[the remark this answers is not in the document]
Comment (frankosgit) [reply src/app/about/page.tsx:49]: good idea

File: src/app/about/page.tsx (hunk 3 above)
Comment (johanehr) [reply src/app/about/page.tsx:60]: "The app is built..." perhaps, so strong preference!

File: src/app/about/page.tsx [hunk 5]
```diff
@@ -8,50 +10,63 @@ export default function About() {
...
         <h1 className="text-2xl mt-6">
-          Hjälp till
+          Join our effort
         </h1>
 
         <p className="mt-2">
-          Vill du hjälpa till att förbättra tjänsten,
-          antingen som utvecklare eller genom att lägga till fler områden, kanske även utanför Bergshamra?
-          Tveka inte att kontakta mig på johanehrenfors@hotmail.com!
+          Would you like to help enhance our service, either by contributing as a developer or by adding new locations, possibly even beyond Bergshamra? Please feel free to contact me at johanehrenfors@hotmail.com!
```
Comment (johanehr) [inline src/app/about/page.tsx:67]: mailto: link?

File: src/app/about/page.tsx [hunk 6]
```diff
@@ -8,50 +10,63 @@ export default function About() {
...
         </p>
 
         <p className="mt-2">
-          Som många andra, så har vi glömt att flytta bilen inför en städdag, vilket har resulterat i saftiga böter.
-          Vi ville snabbt kunna ta reda på vart vi kan låta bilen stå så länge som möjligt, men även ett smidigt sätt att få relevanta påminnelser när det är dags att flytta bilen igen.
+          Like many others, we&apos;ve forgotten to move our car before a cleaning day, resulting in hefty fines.
+          We wanted to quickly find out where we could leave our car for as long as possible, as well as a convenient way to receive relevant reminders when it&apos;s time to move our car again.
         </p>
 
+        {/* notifikationer kommer bli klara när vi lanserad detta   <p className="mt-2">
```
Comment (johanehr) [inline src/app/about/page.tsx:42]: Still in Swedish

Comment (frankosgit) [reply src/app/about/page.tsx:42]: I think we can get rid of this paragraph, thoughts?

Comment (johanehr) [reply src/app/about/page.tsx:42]: Sure!

**Review by johanehr — COMMENTED** [review]
Since I created the PR to see the diff, I can't approve it myself - so I'll let you have the honors, @frankosgit  ;)

**Review by frankosgit — APPROVED** [review]
Great!
